src/Microsoft.Health.CosmosDb/Features/Storage/RetryExceptionPolicyFactory.cs

```diff
@@ -21,19 +22,20 @@ public RetryExceptionPolicyFactory(CosmosDataStoreConfiguration configuration)
             EnsureArg.IsNotNull(configuration, nameof(configuration));
 
             _maxNumberOfRetries = configuration.RetryOptions.MaxNumberOfRetries;
-            _minWaitTime = TimeSpan.FromSeconds(Math.Min(RetryStrategy.DefaultMinBackoff.TotalSeconds, configuration.RetryOptions.MaxWaitTimeInSeconds));
+            _minWaitTime = TimeSpan.FromSeconds(configuration.RetryOptions.MinWaitTimeInSeconds);
             _maxWaitTime = TimeSpan.FromSeconds(configuration.RetryOptions.MaxWaitTimeInSeconds);
         }
 
         public RetryPolicy CreateRetryPolicy()
         {
-            var strategy = new ExponentialBackoff(
-                _maxNumberOfRetries,
-                _minWaitTime,
-                _maxWaitTime,
-                RetryStrategy.DefaultClientBackoff);
+            var policy = Policy
+                .Handle<Exception>(RetryExceptionPolicy.IsTransient)
```

Comment:
We could probably take more advantage of Polly here and handle `DocumentClientException` specifically.